src/sass/partials/_products.scss

```diff
@@ -0,0 +1,215 @@
+.products {
+    font-family: $fontSecondary;
+
+    &__header {
+        color:  $IIIColor;
+        font-size: 34px;
+        text-align: center;
+        text-transform: uppercase;
+    }
+
+    &__label {
+        color: $IVColor;
+        font-size: 18px;
+        margin-top: 121px;
+        margin-bottom: 15px;
+        display: block;
+        text-align: center;
+        text-transform: uppercase;
+    }
+
+    &__container {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: row;
+        gap: 130px;
+        max-width: 280px;
+        margin: 144px auto 98px auto;
+    }
+
+    &__card {
+        width: 100%;
+        height: 485px;
```

Comment:
Try replacing this height by adding with a modifier to the margin-bottom button.

Reply:
I couldn't do it with a modifier - I gave a new class "products__button" and replaced the height with margin-bottom in this button.